docs/source/multidc/index.rst

```diff
@@ -5,21 +5,14 @@ Deploying multi-datacenter ScyllaDB clusters in Kubernetes
 Prepare a platform for a multi datacenter ScyllaDB cluster deployment:
 
 .. toctree::
-   :hidden:
    :maxdepth: 2
 
    eks
    gke
 
-* :doc:`Build multiple Amazon EKS clusters with Inter-Kubernetes networking <eks>`
-* :doc:`Build multiple GKE clusters with Inter-Kubernetes networking <gke>`
-
 Deploy a multi-datacenter ScyllaDB cluster in Kubernetes:
 
 .. toctree::
-   :hidden:
```

Comment:
let's also lower maxdepth to 1 to align with what it looked like before

Reply:
done